taskchampion/taskchampion/src/task/task.rs

```diff
@@ -458,6 +464,14 @@ impl<'r> TaskMut<'r> {
         self.set_string(format!("annotation_{}", entry.timestamp()), None)
     }
 
+    pub fn add_due(&mut self, due: Timestamp) -> Result<()> {
+        self.set_timestamp(Prop::Due.as_ref(), Some(due))
+    }
+
+    pub fn remove_due(&mut self) -> Result<()> {
+        self.set_string(Prop::Due.as_ref(), None)
+    }
```

Comment:
To parallel `set_entry` and `set_wait`, let's have this take an `Option<Timestamp>` and set or remove the value depending on whether that's Some or None.

Reply:
great suggestion! code updated